client/src/ui/track/StageColumn.tsx

```diff
@@ -10,6 +10,7 @@ export interface StageColumnProps {
   stageColor: string;
   applications: Application[];
   handleOpenModal?: (stage: ApplicationStage) => void;
+  onCardDoubleClick?: (application: Application) => void;
```

Comment:
Can we make the card editable in the first click? I would have never known you can edit if I didn't accidentally double click

Reply:
I do double click because user might mistake with one click when they want to drag. Think it as when you want to access a folder, you alway double click to get inside of it right?

Reply:
It's not obvious that it can be double clicked because it's they don't have that context. I would highly recommend to make it a single click